streamer/src/nonblocking/quic.rs

```diff
@@ -85,6 +83,9 @@ const CONNECTION_CLOSE_REASON_TOO_MANY: &[u8] = b"too_many";
 const CONNECTION_CLOSE_CODE_INVALID_STREAM: u32 = 5;
 const CONNECTION_CLOSE_REASON_INVALID_STREAM: &[u8] = b"invalid_stream";
 
+/// Minimal guaranteed lifetime for an unstaked connection
+const UNSTAKED_CONNECTION_LIFETIME: Duration = Duration::from_millis(500);
```

Comment:
In the PR description you sometimes write 400ms, and sometimes 500ms. What could be a criteria to determine optimal lifetime?